tests/src/Search/Api/Response/VorArticleResponseTest.php

```diff
@@ -28,7 +28,6 @@ public function jsonProvider() : array
                     "version": 1,
                     "elocationId": "e09560",
                     "doi": "10.7554\/eLife.09560",
-                    "authorLine": "Lee R Berger et al",
```

Comment:
I believe the authorLine field is now optional, so it doesn't appear in the minimal JSON (an update to the RAML changed the fixtures we test against)